ClinGen/clincoded
# Connects to #1439. VCI's PAGE data table.

**Steps to test:**
1. Go to the VCI and add the **13685** and/or **130281** ClinVar IDs.
2. Upon being taken to the **evidence-only** view of the VCI, click on the **Population** tab.
3. Scroll down the page and expect to see the **PAGE** data table with data correlating to the 11 super populations.
4. Click on the **PAGE Study** anchor link and expect to be shown the **PAGE** credit at the bottom of the tab.
5. Click on the **GGV Browser** link and expect to be taken to the external GGV Browser site.
6. Now try adding a different ID -- **CA10824140** CA ID -- in the VCI.
7. Proceed to the **Population** tab and expect to see _NO_ data being returned in the PAGE data table but a "No population data was found for this allele in PAGE. Search GGV for this variant." message being shown.
